srcpkgs/lsp-plugins/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'lsp-plugins'
+pkgname=lsp-plugins
+version=1.1.30
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkg-config php"
+makedepends="libsndfile-devel libX11-devel libglvnd-devel lv2 cairo-devel ladspa-sdk jack-devel"
+short_desc="Collection of free plugins compatible with LADSPA, LV2 and LinuxVST"
+maintainer="Artur Sinila <freesoftware@logarithmus.dev>"
+license="LGPL-3.0-or-later"
+homepage="https://lsp-plug.in/"
+distfiles="https://github.com/sadko4u/lsp-plugins/archive/refs/tags/${version}.tar.gz"
+checksum=9cf43257729093c240375b3640b1514dff34b092b83b54a5ee68d7e8565c8f80
+
+_arch=$(echo $XBPS_TARGET_MACHINE | sed "s/-musl$//")
+case "$_arch" in
+	i686) export BUILD_PROFILE=i586 ;;
+	*)    export BUILD_PROFILE=${_arch} ;;
+esac
+
+# https://github.com/sadko4u/lsp-plugins/issues/191#issuecomment-812847630
+CXXFLAGS="$CXXFLAGS -std=c++98"
+
+do_check() {
+	export CXXFLAGS="$CXXFLAGS -DLSP_TESTING -DLSP_TRACE"
+	make test
```

Comment:
I think it would be nice to avoid exporting CXXFLAGS with `CXXFLAGS="..." make test` or even override the variable in the Makefile with `make test CXXFLAGS="..."`

Reply:
> I think it would be nice to avoid exporting CXXFLAGS with `CXXFLAGS="..." make test` or even override the variable in the Makefile with `make test CXXFLAGS="..."`

I'm afraid it's not possible https://github.com/void-linux/void-packages/blob/0da27cc76d0765677b4f7dc2c8ce3937489bfe3b/srcpkgs/lsp-plugins/patches/00-makefile-test-remove-cflags.patch

Reply:
If it works with exporting CXXFLAGS, I think it has to work. Did you test it?